app/view/state/__init__.py

```diff
@@ -59,11 +63,13 @@ def updateState():
     population=data.get("population")
     
     result=state.update_State(id,population)
-
+    
     if result:
-        return jsonify({"message": "state updated successfully", "status": 201,"result":result})
+        response=make_response({"message": "state updated successfully", "status": 202,"result":result},202)
```

Comment:
Why `202`?

Reply:
i used it to show that the update has been accepted

Reply:
The 202 Accepted status code means that the request has been accepted for processing, but the _processing has not been finished yet_